chainer/reporter.py

```diff
@@ -356,3 +363,16 @@ def make_statistics(self):
             stats[name + '.std'] = std
 
         return stats
+
+    def serialize(self, serializer):
+        if isinstance(serializer, serializer_module.Serializer):
+            serializer('_names', json.dumps(list(self._summaries.keys())))
+            for name, summary in six.iteritems(self._summaries):
+                if hasattr(summary, 'serialize'):
```

Comment:
Is this line needed?
I think we can always assume that `summary` is a `Summary` instance.

Reply:
I was not sure that each item of `_summaries` is an instance of `Summary`. I will remove that line.